File: BEST-App-main/lib/model/message_recipient.dart
class MessageRecipient {
  final int id;
  final int messageId;
  final int userId;

  MessageRecipient(this.id, this.messageId, this.userId);

  static Map<String, dynamic> toJson(MessageRecipient ob) {
    return {
      'id': ob.id,
      'message_id': ob.messageId,
      'user_id': ob.userId
    };
  }
}

File: spring-best/src/main/java/com/best/springbest/model/MessageRecipient.java
package com.best.springbest.model;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;
import org.springframework.format.annotation.DateTimeFormat;
import javax.persistence.*;
import java.util.Date;

@Entity
@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@Table(name = "message_recipient")
public class MessageRecipient {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @ManyToOne(fetch = FetchType.EAGER, optional = false)
    @JoinColumn(name = "message_id",nullable = false)
    private Message message;

    @ManyToOne(fetch = FetchType.EAGER, optional = false)
    @JoinColumn(name = "user_id",nullable = false)
    private User user;
}
